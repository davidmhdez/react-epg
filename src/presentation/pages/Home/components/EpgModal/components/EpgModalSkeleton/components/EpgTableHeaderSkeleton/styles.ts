import styled from "styled-components";

export const EpgTableHeaderSkeletonWrapper = styled.div`
    width: 100%;
    display: flex;
    padding: 10px 0;
    overflow: hidden;
    
    .dayText{
        width: 30px;
        height: 15px;
        margin: 0 90px;
    }
    
    .hourText{
        width: 60px;
        height: 15px;
        margin: 0 70px;
    }
`