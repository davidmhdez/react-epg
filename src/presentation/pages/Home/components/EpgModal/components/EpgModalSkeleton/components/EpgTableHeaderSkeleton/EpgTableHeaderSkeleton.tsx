import {EpgTableHeaderSkeletonWrapper} from "./styles.ts";
import Skeleton from "react-loading-skeleton";

const EpgTableHeaderSkeleton = () => {
    return (
        <EpgTableHeaderSkeletonWrapper>
            <Skeleton className="dayText"/>
            <Skeleton className="hourText"/>
            <Skeleton className="hourText"/>
            <Skeleton className="hourText"/>
            <Skeleton className="hourText"/>
            <Skeleton className="hourText"/>
            <Skeleton className="hourText"/>
        </EpgTableHeaderSkeletonWrapper>
    );
};

export default EpgTableHeaderSkeleton;
