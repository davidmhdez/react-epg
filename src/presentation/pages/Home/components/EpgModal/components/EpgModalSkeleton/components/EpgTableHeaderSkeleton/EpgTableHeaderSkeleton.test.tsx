import EpgTableHeaderSkeleton from "./EpgTableHeaderSkeleton.tsx";
import {render} from "@testing-library/react";

describe('EpgTableHeaderSkeleton', () => {
    test('should match with snapshot', () => {
        const { container } = render(<EpgTableHeaderSkeleton />);
        expect(container).toMatchSnapshot();
    })
})