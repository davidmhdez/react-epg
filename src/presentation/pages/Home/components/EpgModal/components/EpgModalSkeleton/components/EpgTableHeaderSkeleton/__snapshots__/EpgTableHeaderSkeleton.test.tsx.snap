// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EpgTableHeaderSkeleton should match with snapshot 1`] = `
<div>
  <div
    class="sc-aYaIB cjIWLn"
  >
    <span
      aria-busy="true"
      aria-live="polite"
    >
      <span
        class="react-loading-skeleton dayText"
      >
        ‌
      </span>
      <br />
    </span>
    <span
      aria-busy="true"
      aria-live="polite"
    >
      <span
        class="react-loading-skeleton hourText"
      >
        ‌
      </span>
      <br />
    </span>
    <span
      aria-busy="true"
      aria-live="polite"
    >
      <span
        class="react-loading-skeleton hourText"
      >
        ‌
      </span>
      <br />
    </span>
    <span
      aria-busy="true"
      aria-live="polite"
    >
      <span
        class="react-loading-skeleton hourText"
      >
        ‌
      </span>
      <br />
    </span>
    <span
      aria-busy="true"
      aria-live="polite"
    >
      <span
        class="react-loading-skeleton hourText"
      >
        ‌
      </span>
      <br />
    </span>
    <span
      aria-busy="true"
      aria-live="polite"
    >
      <span
        class="react-loading-skeleton hourText"
      >
        ‌
      </span>
      <br />
    </span>
    <span
      aria-busy="true"
      aria-live="polite"
    >
      <span
        class="react-loading-skeleton hourText"
      >
        ‌
      </span>
      <br />
    </span>
  </div>
</div>
`;
